public/main.js

```diff
@@ -8,23 +8,28 @@ var deleteComplete = document.getElementById("delete-complete");
 var all = document.getElementById("all");
 var active = document.getElementById("active");
 var completed = document.getElementById("completed");
+var displayMessage = document.getElementById("display-message");
 var ALL_VALUES = 0;
 var ACTIVE_VALUES = 1;
 var COMPLETE_VALUES = 2;
 var mode = ALL_VALUES;
+displayMessage.innerHTML = "Displaying all todo items.";
 
 all.onclick = function() {
     mode = ALL_VALUES;
+    displayMessage.innerHTML = "Displaying all todo items.";
```

Comment:
This text could get out of sync with above so another variable would probably work well here